packages/moonstone/TooltipDecorator/util.js

```diff
@@ -144,12 +157,37 @@ const getPosition = function (tooltipNode, clientNode, arrowAnchor, tooltipDirec
 		}
 	}
 
+	// When tooltip is too wide, shift the tooltip so that the first part of the tooltip is always visible. Does not affect tooltips with `tooltipDirection` of `left` and `right`
+	if (overflow.isOverWide && !(tooltipDirection === 'left' || tooltipDirection === 'right') && arrowAnchor !== 'right') {
+		position.left = rtl ? window.innerWidth - tooltipNode.width : 12;
+	}
+
 	return position;
 };
 
+/**
+ * Adjusts the `Tooltip` arrow anchor when the tooltip is too wide.
+ * Takes the output of `calcOverflow`.
+ *
+ * @method
+ * @memberof moonstone/TooltipDecorator
+ * @param   {Object}  clientNode		The `getBoundingClientRect` values for client node
```

Comment:
It seems nitpicky, but we should probably use spaces to align these since the variable width of tabs can mess up the alignment if it's not consistently set everywhere. There was another section above that did the same thing.

Reply:
FWIW, we use tabs a good bit in the docs.